src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -0,0 +1,192 @@
+<?php
+namespace LORIS\Middleware;
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \Psr\Http\Server\{MiddlewareInterface, RequestHandlerInterface};
+
+class UserPageDecorationMiddleware implements MiddlewareInterface {
+    protected $JSFiles;
+    protected $CSSFiles;
+    protected $Config;
+    protected $BaseURL;
+    protected $PageName;
+
+    public function __construct(\User $user, string $baseurl, string $pagename, \NDB_Config $config, array $JS, array $CSS) {
+        $this->JSFiles = $JS;
+        $this->CSSFiles = $CSS;
+        $this->Config = $config;
+        $this->BaseURL = $baseurl;
+        $this->PageName = $pagename;
+        $this->user = $user;
+    }
+
+    /**
+     * Displays a page to an anonymous (not logged in) user.
```

Comment:
logged in user (not anonymous)

Reply:
thanks.. I noticed that doc comments were also wrong, so I fixed them at the same time.